### PR TITLE
Remove a message from stderr

### PR DESCRIPTION
**Pull request by jan-cerny**
When a remediation can't find the respective rule YML, it produces
a message on stderr, which pollutes the build output. In fact, this
is a normal situation when the remediation exists but the rule
isn't a part of the given product.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by matusmarhefka** [comment]
@jan-cerny Can we rather disable the error only in case that the rule isn't a part of the given product and leave the error in other cases?

**Comment by jan-cerny** [comment]
@matusmarhefka I have changed it to check the existence in the caller instead. If user uses the method in its own code with a nonexistent path it will hopefully raise him an exception. I also removed the unused code that I noticed.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

File: ssg/build_remediations.py [hunk 1]
```diff
@@ -258,10 +246,6 @@ class BashRemediation(Remediation):
     def __init__(self, file_path):
         super(BashRemediation, self).__init__(file_path, "bash")
 
-    def load_associated_rule(self, resolved_rules_dir, rule_id):
-        # No point in loading rule for this remediation type as of now
-        pass
-
 
 class AnsibleRemediation(Remediation):
```
Comment (matusmarhefka) [inline ssg/build_remediations.py]: Add `isfile` check also into the `from_snippet_and_rule` method of this class.

Comment (jan-cerny) [reply ssg/build_remediations.py]: OK, I will do that.

Comment (jan-cerny) [reply ssg/build_remediations.py]: done

**Comment by matusmarhefka** [comment]
@jan-cerny LGTM, thanks.

**Pull request merged**